src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions.mdx

```diff
@@ -16,9 +16,9 @@ redirects:
   - /docs/alerts/new-relic-alerts/defining-conditions/create-alert-conditions-nrql-queries
 ---
 
-import queriesPolicyNewAlertCondition from 'images/queries-nrql_screenshot-crop_create-alert-condition.webp'
+import queriesPolicyNewAlertCondition from 'images/queries-nrql_screenshot_create-alert-condition.webp'
```

Comment:
This filename should be as follows: queries-nrql_screenshot-crop_create-alert-condition.webp. You need to indicate if is full or crop. You have [here](https://docs.newrelic.com/docs/style-guide/images/embed-images/#image-name-convention) the explanation.